### ISSUE TITLE
screen flicker on restore in Windows after Electron 6 upgrade

### ISSUE THREAD

**Issue by whyboris** [whyboris/Video-Hub-App#260]
I upgraded to Electron 6 with #257 

Now, on Windows, whenever you restore the window from minimized state, it flickers once or twice before showing the content. This is unpleasant, especially if in _dark mode_ 😖 

I'm investigating today 🤞

**Comment by whyboris** [comment]
Current best guess: the window seems to change to some very small size just before restoring, and returns to original size - requiring for the whole app to re-compute and re-paint the entire UI 😠 

So far have been unsuccessful with saving and restoring the window size manually. Tried this:
```ts
  let widthHack: number;
  let heightHack: number;

  win.on('minimize', (e) => {
    const bounds: any = win.getBounds();
    widthHack = bounds.x;
    heightHack = bounds.y;
  });

  win.on('restore', (e) => {
    win.setSize(widthHack, heightHack);
  }
```

**Comment by whyboris** [comment]
I opened a ticket with _Electron_ as the issue is definitely with the restoring window size:
https://github.com/electron/electron/issues/19816

The flicker does _not_ happen when the window is maximized, minimized, and restored. Only if the window was not maximized, minimized, and restored 🤷‍♂

**Comment by whyboris** [comment]
Looks like it will be fixed with https://github.com/electron/electron/pull/19928 🎉 ❤️ 🙇 
I love _Electron_ and its maintainers 🙇

**Comment by whyboris** [comment]
😢 Is not fixed as of `6.0.5` - https://github.com/electron/electron/issues/19816#issuecomment-526161552

**Comment by whyboris** [comment]
Turns out it will be released with Electron `6.1.0` release 🎉  https://github.com/electron/electron/issues/19816#issuecomment-526423893

**Comment by whyboris** [comment]
Closed with a push to _master_ https://github.com/whyboris/Video-Hub-App/commit/309c1e7c6deb71fb0e14e422fa311c722db220e7 🎉
